Userland/Libraries/LibJS/Runtime/Realm.cpp

```diff
@@ -51,6 +51,10 @@ ThrowCompletionOr<NonnullOwnPtr<ExecutionContext>> Realm::initialize_host_define
     // 5. Set the ScriptOrModule of newContext to null.
     new_context->script_or_module = {};
 
+    // NOTE: This is not in the spec.
+    static DeprecatedFlyString realm_execution_context_name = "(realm execution context)";
+    new_context->function_name = realm_execution_context_name;
+
```

Comment:
Not sure about this one, feels like we should just start hiding the bottom one from stack traces